### PR TITLE
<Dropdown/> - change cursor type to pointer on input field

### PR DESCRIPTION
**Pull request by daugerdas**
### 🔦 Summary
Change cursor type to `pointer` from `text` on `<Dropdown/>`'s input field

Description: [DSM-742](https://jira.wixpress.com/browse/DSM-742)

This change aims to have a unit test for cursor type, but currently having an issue with receiving the correct element style for assertion and writing a test that runs both on `[sync]` and `[async]` drivers.

...

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6138

File: src/Dropdown/test/Dropdown.spec.js [hunk 1]
```diff
@@ -275,5 +275,13 @@ describe('Dropdown', () => {
...
       });
     });
+
+    it('should have cursor: "pointer" on the input', async () => {
+      const { driver } = createDriver(<Dropdown options={getOptions()} />);
+      const input = (await driver.inputWrapper()).querySelector(
+        '[data-hook="wsr-input"]',
+      );
+      expect(window.getComputedStyle(input).cursor).toEqual('pointer');
+    });
```
Comment (daugerdas) [inline src/Dropdown/test/Dropdown.spec.js:285]: different version for comparison with `.element()` that seems to work only on `[sync]` driver for me
```suggestion
    it('should have cursor: "pointer" on the input', async () => {
      const { inputDriver } = createDriver(<Dropdown options={getOptions()} />);
      const input = (await inputDriver.element()).querySelector(
        '[data-hook="wsr-input"]',
      );
      expect(window.getComputedStyle(input).cursor).toEqual('pointer');
    });
```

**Comment by argshook** [comment]
@daugerdas please see https://github.com/wix/wix-style-react/pull/6140

**Review by argshook — APPROVED** [review]

**Comment by argshook** [comment]
unrelated visual tests failing, otherwise good to merge. Great work! :+1:

**Pull request merged**